spec/rubocop/cop/metrics/utils/code_length_calculator_spec.rb

```diff
@@ -467,9 +467,12 @@ def test
         end
       RUBY
 
+      warning = 'Unknown foldable type: :send. ' \
+                'Valid foldable types are: array, hash, heredoc, method_call.'
```

Comment:
Would it be better to standardize the notation to symbols, in line with `:send`?

```suggestion
                'Valid foldable types are: :array, :hash, :heredoc, :method_call.'
```

Reply:
I think no (and probably not for `:send` either), because these values are strings set in the `CountAsOne` config for `Metrics/BlockLength`, etc. 

But actually, that probably indicates that this check is happening in the wrong place because “foldable types” is a concept that the user doesn’t directly engage with.